libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeViewRenderer/TreeViewNode.tsx

```diff
@@ -33,13 +33,23 @@ export class TreeViewNode extends React.Component<ITreeViewNodeProps> {
     const classNames = treeViewRendererStyles();
     const gradientFillId = getRandomId();
     const theme = getTheme();
+    const nodeFilterProps = node.data.filterProps;
+    const ariaLabel = `${nodeFilterProps.numCorrect} ${
+      nodeFilterProps.numIncorrect
+    } ${nodeFilterProps.errorCoverage.toFixed(
+      2
+    )} ${nodeFilterProps.metricValue.toFixed(2)}`; // TODO: waiting for PM input
```

Comment:
This ariaLabel string is temporary, waiting for PM's input on what the screen reader should announce when traversing each tree node